samples/ClassroomLabs/Modules/Library/Tests/RunPesterTests.ps1

```diff
@@ -4,25 +4,20 @@ Param(
 
     [Parameter(Mandatory = $false, HelpMessage="If we should run the tests in 'verbose' mode for extra logging")]
     [bool] $VerboseTests = $false
-
 )
 
-# Import the module here to make sure we validate up front versions of Azure Powershell
 Import-Module $PSScriptRoot\..\Az.LabServices.psm1
 
-# We don't want to give up on the rest after a single error
-$ErrorActionPreference="Continue"
+# This allows the parallel tests not to give up on all of them when one test fails. Using just 'Continue' doesn't do it. Mystery.
+$ErrorActionPreference="SilentlyContinue"
 
-# Check if we have a newer version of Pester, if not - let's install it
```

Comment:
I looks like the existing comments were removed?  Just wondering if that was intentional?

Reply:
Yes. I understood the logic just glancing at the code. So removed them to make the code more concise.